src/mixs/schema/mixs.yaml

```diff
@@ -9628,15 +9628,15 @@ slots:
     slot_uri: MIXS:0000283
     multivalued: true
     range: string
+    deprecated: true, replace with MIXS:0000270 https://github.com/GenomicsStandardsConsortium/mixs/issues/743 
   nose_throat_disord:
-    description: History of nose-throat disorders; can include multiple disorders,  The
-      terms should be chosen from the DO (Human Disease Ontology) at http://www.disease-ontology.org,
-      lung disease (https://disease-ontology.org/?id=DOID:850), upper respiratory
-      tract disease (https://disease-ontology.org/?id=DOID:974)
+    description: Report any history of nose, mouth, teeth and/or throat disorders in the subject. May include multiple disorders.
     title: nose throat disorder
     keywords:
       - disorder
     slot_uri: MIXS:0000270
+    aliases: nose_mouth_teeth_throat_disord
+    comments: "The terms should be chosen from the DO (Human Disease Ontology) at http://www.disease-ontology.org, lung disease (https://disease-ontology.org/?id=DOID:850), upper respiratory tract disease (https://disease-ontology.org/?id=DOID:974)"
```

Comment:
comments take a list, not a scalar

Reply:
I fixed the scalar comment

I also followed the block/folded/clip style at https://yaml-multiline.info/. That's "better"/more explicit but may not be consistent with the rest of mixs.yaml. So we should go though other long values and follow that same style at some point.